README.md

```diff
@@ -97,10 +98,11 @@
 - [CloudCheckr](https://www.cloudcheckr.com/) - Cloud management platform for cost, security, and compliance. Tags: cost_visibility, multi_cloud
 - [CloudHealth](https://cloudhealth.vmware.com/) - Cloud cost visibility and automated optimizations. Tags: cost_visibility, multi_cloud
 - [CloudPouch](https://cloudpouch.dev/) - A unique desktop application that works with any AWS account, uses AWS CLI proviles. Cloud cost visibility and optimizations. Tags: cost_visibility, automated_optimization, AWS, EBS
-- [CloudThread](https://www.cloudthread.io/) - Cloud cost tracking and waste identification. Tags: cost_visibility, multi_cloud
```

Comment:
the blog link gives a 404 because it misses the last character https://www.finout.io/blog/cloudthread_finou